_layouts/i18n.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="{{ page.language_code }}" dir="{% if page.language_direction %}{{ page.language_direction }}{% else %}ltr{% endif %}">
+
+{% include i18n-head.html %}
+
+<body itemscope="itemscope" itemtype="http://schema.org/WebPage">
+
+{% include i18n-header.html %}
+
+<script type="html/template" id="listing-template">
+<div class="listing-container">
+  <a href="">
+    <table>
+      <tr>
+        <td class="column-1"><i class="fa fa-star-o"></i></td>
+        <td class="column-2">
+          <h3><span class="title-span"></span></h3>
+          <div class="updated-views-div"><span class="updated-span"></span>&emsp;<i class="fa fa-eye"></i> <span class="views-span"></span></div>
+          <div class="author-div"></div>
+          <div class="pinned-to-bottom">
+            <h3><span class="language-title-span"></span></h3>
+            <div class="language-code-div"></div>
+          </div>
+        </td>
+      </tr>
+    </table>
+  </a>
+</div>
+</script>
+
+<script type="html/template" id="more-template">
+  <div class="more-container">
+    <span><a href="" class="bold">{{ site.data[page.language_code].l10n.view_more }}</a></span>
+  </div>
+</script>
+
+<header id="pinned-header" class="pin-to-top">
+  <div class="container">
+    <div class="float-left">
+      <table>
+        <tr>
+          <td><span class="bold">{{ site.data[page.language_code].l10n.popular }}</span>&emsp;<span class="bold">{{ site.data[page.language_code].l10n.recent }}</span></td>
```

Comment:
The Popular and Recent links at the top should link to the respective headings on the page.